docs/src/api.ts

```diff
@@ -53,21 +53,64 @@ async function createError(response: Response): Promise<ApiError> {
 
 export async function renderImage(body: object): Promise<Blob> {
   const requestObject = buildRequestRenderImage(body);
+  return retryRequest(fetchWithTimeout, '/render_image', requestObject);
+}
 
-  const backend_url: URL = new URL(
+async function fetchWithTimeout(
+  path: string,
+  requestObject: RequestObject,
+  timeout: number
+): Promise<Response> {
+  const backendUrl: URL = new URL(
     config.customFields.environment === 'development'
       ? 'http://localhost:8081'
       : 'https://playground.compositor.live'
   );
```

Comment:
```suggestion
  const backendUrl: string = config.customFields.environment === 'development'
      ? 'http://localhost:8081'
      : 'https://playground.compositor.live';
```

you could also move this value to the global const on top of the file